connectors/src/connectors/notion/index.ts

```diff
@@ -38,33 +32,22 @@ import type { NangoConnectionId } from "@connectors/types/nango_connection_id";
 import { BaseConnectorManager } from "../interface";
 import { getParents } from "./lib/parents";
 
-const { getRequiredNangoNotionConnectorId } = notionConfig;
-
 const logger = mainLogger.child({ provider: "notion" });
 
 async function workspaceIdFromConnectionId(connectionId: string) {
-  if (isDualUseOAuthConnectionId(connectionId)) {
-    const tokRes = await getOAuthConnectionAccessToken({
-      config: apiConfig.getOAuthAPIConfig(),
-      logger,
-      provider: "notion",
-      connectionId,
-    });
-    if (tokRes.isErr()) {
-      return new Err("Error retrieving access token");
-    }
-    return new Ok(
-      (tokRes.value.scrubbed_raw_json as { workspace_id?: string })
-        .workspace_id ?? null
-    );
-  } else {
-    const connectionRes = await getConnectionFromNango({
-      connectionId: connectionId,
-      integrationId: getRequiredNangoNotionConnectorId(),
-      refreshToken: false,
-    });
-    return new Ok(connectionRes?.credentials?.raw?.workspace_id || null);
+  const tokRes = await getOAuthConnectionAccessToken({
+    config: apiConfig.getOAuthAPIConfig(),
+    logger,
+    provider: "notion",
+    connectionId,
+  });
+  if (tokRes.isErr()) {
+    return new Err("Error retrieving access token");
```

Comment:
Can we leverage the error returned by `getOAuthConnectionAccessToken`?

Reply:
Added more logging since we can't return it directly